### ISSUE TITLE
Custom Easing in Actions

### ISSUE THREAD

**Issue by dariodepaolis** [cocos2d/cocos2d-x#11665]
I think this is a request because I haven't found this feature reading through docs, wiki and guide.

Is there any way to implement some custom easing actions?

I mean something like this:
http://timotheegroleau.com/Flash/experiments/easing_function_generator.htm5

**Comment by milos1290** [comment]
Looking at the tests here is the example how to use any of the easing actions

<code>
 sprite->runAction(EaseSineInOut::create(MoveBy::create(2, Vec2(200,0))));
</code>

I havent tried it, but it looks like it will work.

**Comment by dariodepaolis** [comment]
No, I mean a custom easing, not the normal EaseIn/Out, EaseExponential, EaseSine, etc
Like a cubic bezier
http://cubic-bezier.com/

**Comment by walzer** [comment]
Currently cocos2d-x haven't supported this feature yet. I know this is an important feature for advanced users.

**Comment by dariodepaolis** [comment]
Ok thank you for interesting.
